### DISCUSSION TITLE
Getting probability-of-precipitation

### DISCUSSION THREAD

**Discussion by cc8080** [weather-gov/api#439] [Q&A]
I'm converting from the old DWML format, but running into a problem because the probability-of-precipitation doesn't exist in the new JSON format.  I see other comments about this but don't see any recent updates about when/whether it will be added.  Please reply with an update. I'm not super excited about trying to cobble together a solution from the raw data. :)

**Comment by JJKraw** [comment]
Which endpoint(s) are you referring to?

POPs are included in /gridpoints/{wfo}/{x},{y}

If at or above a certain threshold (20% maybe?) it will appear in detailedForecast text for a particular period in the hourly endpoint.

**Reply by JJKraw** [reply]
> I'm not super excited about trying to cobble together a solution from the raw data.

Ah, missed this my first read so you probably know what I just typed...
